common/src/sql/dbinit.sql

```diff
@@ -2354,6 +2354,35 @@ CREATE UNIQUE INDEX ON omicron.public.switch_port_settings_group (
 ) WHERE
     time_deleted IS NULL;
 
+CREATE SEQUENCE nat_gen START 1 INCREMENT 1;
+
+CREATE TABLE omicron.public.ipv4_nat_entry (
+    id UUID PRIMARY KEY,
+    external_address INET NOT NULL,
+    first_port INT4 NOT NULL,
+    last_port INT4 NOT NULL,
+    sled_address INET NOT NULL,
+    vni INT4 NOT NULL,
+    mac INT8 NOT NULL,
+    gen INT8 NOT NULL DEFAULT nextval('nat_gen') ON UPDATE nextval('nat_gen'),
+    time_created TIMESTAMPTZ NOT NULL DEFAULT now(),
+    time_deleted TIMESTAMPTZ
+);
+
+CREATE UNIQUE INDEX ON omicron.public.ipv4_nat_entry (
+    gen
+)
+STORING (
+    external_address,
+    first_port,
+    last_port,
+    sled_address,
+    vni,
+    mac,
+    time_created,
+    time_deleted
+);
```

Comment:
As a PSA:

- This file is now in `schema/crdb/dbinit.sql`. You should be able to update that file directly, and keep using this more-or-less as-is.
- I recommend adding a new "rev" of CRDB (3.0.0? Or whatever the +1 of the major version would be), and including "just the addition of this sequence + table + index" within a file named `up.sql`
- It's possible to make these calls idempotent! The `IF NOT EXISTS` keywords will help.

Reply:
Yeah, I need to rebase and use the new mechanism for updating the schema :D